061b045

commit 061b045
Author: Athanatos123 <alain.delree@gmail.com>
Date:   Fri Jul 31 10:36:55 2026 +0200

    Remet en evidemence le point Bouton Interrompre dans l'onglet CCW

diff --git a/TACHES.md b/TACHES.md
index 132b5b5..d5e19bb 100644
--- a/TACHES.md
+++ b/TACHES.md
@@ -4,6 +4,7 @@ Idées et pistes non prioritaires, à réaliser éventuellement plus tard.
 Alain peut modifier ce fichier directement, sans passer par une issue.
 
 ---
+
 ##Bouton Interrompre dans l'onglet CCW
 
 Procédure : nssm restart CCW-Watcher, puis supprimer le(s) fichier(s)
